### DISCUSSION TITLE
Edit list members

### DISCUSSION THREAD

**Discussion by Gd-Gordon** [robertodoering/harpy#611] [Ideas]
I cannot find how to add a twitter user to my lists instead of following them. I also cannot edit my existing list members. This feature is common in many other clients and is very useful to manage/filter lots of users. Hope developer can add it as soon as possible or disclose the roadmap/possibility. 
Many thanks to developer for making this fantastic twitter client.

**Comment by robertodoering** [comment]
Thanks, this feature is planned: #280
